lib/watchErrorHandler.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (message, status) =>  {
+  if(status === 0) {
+    console.log(message);
+  } else {
+    console.error( message.stack ? message.stack : String(message));
```

Comment:
A couple small things:
1. Spacing
2. You could use `||` here such that `message.stack || String(message)`
3. I'm not exactly sure what `String(message)` will produce on an object. I've had good luck with `JSON.stringify(message, null, 2)` properly converting an object to a string. Oftentimes `Object.toString()` simply results in "[Object object]".